src/main/java/com/github/manolo8/darkbot/core/manager/PetManager.java

```diff
@@ -11,10 +11,7 @@
 import com.github.manolo8.darkbot.core.objects.Gui;
 import com.github.manolo8.darkbot.core.objects.swf.ObjArray;
 
-import java.util.ArrayList;
-import java.util.Comparator;
-import java.util.List;
-import java.util.Locale;
+import java.util.*;
```

Comment:
Avoid wildcard imports, edit your IDE settings not to use them

Reply:
done